Adding tests for origDatablocks

nitrosx opened the pull request:
## Description
Added explicit testing for endpoint origdatablocks

## Motivation
In the latest code review following issues 638, we realized that there was not any explicit testing for the origdatablocks endpoint.

## Changes:
* added file OrigDatablockForRawDataset.js

## Tests included/Docs Updated?
- [ ] Included for each change/fix?
- [ ] Passing? (Merge will not be approved unless this is checked) 
- [ ] Docs updated?
- [ ] New packages used/requires npm install? 
- [ ] Toggle added for new features?

nitrosx committed:
done coding new tests for origdatablock

nitrosx committed:
Merge branch 'master' of https://github.com/SciCatProject/scicat-backend-next into github-issue-638

nitrosx committed:
Coding done. Started testing

nitrosx committed:
wip to pass the new test on origdatablocks

nitrosx committed:
Added som tests. All tests passing.

nitrosx committed:
Merge branch 'master' into github-issue-638

nitrosx committed:
Fixed linting issues

nitrosx committed:
Merge branch 'github-issue-638' of https://github.com/SciCatProject/scicat-backend-next into github-issue-638

nitrosx committed:
fixed linting issues

nitrosx committed:
Merge branch 'master' of https://github.com/SciCatProject/scicat-backend-next into github-issue-638

nitrosx committed:
fixed some tests and numbered tests in modified files

nitrosx committed:
Fixed tests and linting

nitrosx committed:
fixed tests and updated github workflow configuration

martin-trajanovski reviewed (approved):
Looks good to me in general. The only thing is that this PR doesn't add only the tests for origDatablocks as it is saying in the title but also fixes some bugs I guess. Another thing is with the numbering of the tests. Not really sure if we need that or not.

nitrosx commented:
Originally, this PR was meant to add only the tests, than I found issues with some of the endpoint (related to the tests that I added), so I addressed them here.
Regarding the numbering of the tests, they make it easier to find them in code... at least for me.
